docs/user/README.md

```diff
@@ -86,28 +114,121 @@ AZURE_STORAGE_ACCOUNT_NAME
+
+```shell
+MODELARDBM_PORT=8888
+MODELARDBD_PORT=8889
+```
+
+### Ingest Data
+Before time series can be ingested into `modelardbd`, a model table must be created. From a user's perspective a model
+table functions like any other table and can be queried using SQL. However, the implementation of model table is highly
+optimized for time series and a model table must contain a single column with timestamps, one or more columns with
+fields (measurements as floating-point values), and zero or more columns with tags (metadata as strings). Model tables
+can be created using `CREATE MODEL TABLE` statements with the column types `TIMESTAMP`, `FIELD`, and `TAG`. For `FIELD`
+an error bound can optionally be specified in parentheses to enable lossy compression with a relative per value error
+bound, e.g., `FIELD(1.0)` creates a column with a one percent error bound. `FIELD` columns default to an error bound of
+zero when none is specified. If the values in a `FIELD` column can be computed from other columns they need not be stored.
+Instead, if a `FIELD` column is defined using the syntax `FIELD AS expression`, e.g., `FIELD AS column_one + column_two`,
+the values of the `FIELD` column will be the result of the expression. As these generated `FIELD` columns do not store any
+data, an error bound cannot be defined. `modelardb` also supports normal tables created through `CREATE TABLE` statements.
+
+As both `CREATE MODEL TABLE` and `CREATE TABLE` are just SQL statements, both types of tables can be created using
+`modelardb` or programmatically using Apache Arrow Flight. For example, a model table storing a simple multivariate
+time series with weather data collected at different wind turbines can be created as follows:
 
 ```shell
-MODELARDBD_PORT=9998
+CREATE MODEL TABLE wind_turbine(timestamp TIMESTAMP, wind_turbine TAG, wind_direction FIELD, wind_speed FIELD(1.0))
 ```
 
+The following example shows how to create the same model table in Python using Apache Arrow Flight:
+
+```python
+from pyarrow import flight
+
+flight_client = flight.FlightClient("grpc://127.0.0.1:9999")
+
+sql = "CREATE MODEL TABLE wind_turbine(timestamp TIMESTAMP, wind_turbine TAG, wind_direction FIELD, wind_speed FIELD(1.0))"
+action = flight.Action("CommandStatementUpdate", str.encode(sql))
+result = flight_client.do_action(action)
+
+print(list(result))
+```
+
+When running a larger cluster of `modelardbd` instances, it is required to use `modelardbm` to create tables and model 
+tables. This is a necessity as `modelardbm` is responsible for keeping the database schema consistent across all 
+`modelardbd` instances in the cluster. The process for creating a table on the manager is the same as when creating the 
+table directly on a `modelardbd` instance, as shown above. The only difference is that the gRPC URL should be changed to 
+connect to the flight client of the manager instead. When the table is created through `modelardbm`, the table is 
```

Comment:
Is the manager not a flight server and not a client? Maybe just write something like "... should be changed to connect to the manager instead."